src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,77 @@
+package duke.parser;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
```

Comment:
Perhaps there should be a line break before `import org.junit.jupiter.api.Test;` ? Please do check on this as I am not sure about this as well.

Reply:
You're right, we should separate related imports. I will change this, thanks for noticing 😄